blocks/math.ts

```diff
@@ -500,20 +491,17 @@ const IS_DIVISIBLEBY_MUTATOR_MIXIN = {
  * 'math_is_divisibleby_mutator' extension to the 'math_property' block that
  * can update the block shape (add/remove divisor input) based on whether
  * property is "divisible by".
- * @this {Block}
- * @package
  */
-const IS_DIVISIBLE_MUTATOR_EXTENSION = function() {
-  this.getField('PROPERTY')
-      .setValidator(
-          /**
-           * @this {FieldDropdown}
-           * @param {*} option The selected dropdown option.
-           */
-          function(option) {
-            const divisorInput = (option === 'DIVISIBLE_BY');
-            this.getSourceBlock().updateShape_(divisorInput);
-          });
+const IS_DIVISIBLE_MUTATOR_EXTENSION = function(this: Block) {
+  this.getField('PROPERTY')!.setValidator(
+      /**
+       * @param option The selected dropdown option.
+       */
+      function(this: FieldDropdown, option) {
```

Comment:
Add a type for `option`?

Reply:
Hmm.  It was type `{*}` before, so I guess `any`.  Or do you think `AnyDuringMigration` would be a better choice for now?

Reply:
(I'm a little confused about where `option` comes from.  `.setValidator` takes a `FieldValidator`, which is just a one-argument function.…)

Reply:
Ahh, no: `FieldDropdown extends Field<string>`, and `.setValidator` takes a `FieldValidator<T>`, which is a `(T) => T | null | undefined`.  So `option` here is a `string`.

Reply:
Can the type be added accordingly?